feconf.py

```diff
@@ -75,6 +75,11 @@
     'TextInput': 'LDAStringClassifier'
 }
 
+# A mapping of classifier algorithms to their current version numbers.
```

Comment:
It is not classifier version per se, it is more like classifier schema data version - to keep track of the version of the dict stored in the classifier object.